### ISSUE TITLE
Fail to install serch-services

### ISSUE THREAD

**Issue by silverskyvicto** [Alfresco/SearchServices#207]
Hi.

II tried installing the Search-services project.However, it has failed.
The situation is as follows.

### Subject
An error occurs due to lack of dependencies.

### Performed steps

1. Clone project using `git` command below.
    ```
    $ git clone git@github.com:Alfresco/SearchServices.git
    ```
2. Install using `mvn` command below.
    ```
    $ cd SearchServices/search-services
    $ mvn clean install
    ```
#### Expected
Build suceeded.

#### Actual
Error occured with error messages below.
```
[INFO] --------------------< org.alfresco:alfresco-search >--------------------
[INFO] Building Alfresco Solr Search 1.5.0-SNAPSHOT                       [3/4]
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for org.apache.lucene:lucene-analyzers-common:jar:6.6.5-patched is missing, no dependency information available
[WARNING] The POM for org.apache.solr:solr-core:jar:6.6.5-patched is missing, no dependency information available
[WARNING] The POM for org.apache.solr:solr-analysis-extras:jar:6.6.5-patched is missing, no dependency information available
[WARNING] The POM for org.apache.solr:solr-langid:jar:6.6.5-patched is missing, no dependency information available
[WARNING] The POM for org.apache.solr:solr-clustering:jar:6.6.5-patched is missing, no dependency information available
[WARNING] The POM for org.apache.solr:solr-test-framework:jar:6.6.5-patched is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Alfresco Solr Search parent 1.5.0-SNAPSHOT:
[INFO] 
[INFO] Alfresco Solr Search parent ........................ SUCCESS [  0.255 s]
[INFO] Alfresco Solr Client ............................... SUCCESS [  4.462 s]
[INFO] Alfresco Solr Search ............................... FAILURE [  0.108 s]
[INFO] Alfresco Search Services ........................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.936 s
[INFO] Finished at: 2019-10-14T16:53:31+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project alfresco-search: Could not resolve dependencies for project org.alfresco:alfresco-search:jar:1.5.0-SNAPSHOT: The following artifacts could not be resolved: org.apache.lucene:lucene-analyzers-common:jar:6.6.5-patched, org.apache.solr:solr-core:jar:6.6.5-patched, org.apache.solr:solr-analysis-extras:jar:6.6.5-patched, org.apache.solr:solr-langid:jar:6.6.5-patched, org.apache.solr:solr-clustering:jar:6.6.5-patched, org.apache.solr:solr-test-framework:jar:6.6.5-patched: Failure to find org.apache.lucene:lucene-analyzers-common:jar:6.6.5-patched in https://artifacts.alfresco.com/nexus/content/groups/public was cached in the local repository, resolution will not be reattempted until the update interval of alfresco-public has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :alfresco-search
```

### Environment Info
```
$ java -version
openjdk version "13" 2019-09-17
OpenJDK Runtime Environment (build 13+33)
OpenJDK 64-Bit Server VM (build 13+33, mixed mode, sharing)
$ mvn -v
Apache Maven 3.6.2 (40f52333136460af0dc0d7232c0dc0bcf0d9e117; 2019-08-28T00:06:16+09:00)
Maven home: /usr/local/Cellar/maven/3.6.2/libexec
Java version: 13, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-13.jdk/Contents/Home
Default locale: ja_JP, platform encoding: UTF-8
OS name: "mac os x", version: "10.15", arch: "x86_64", family: "mac"
```

### Other Info
In Alfresco Nexus Repository, There seems to be no "org.apache.lucene:lucene-analyzers-common: jar:6.6.5-patched" in the repository.

**Comment by aborroy** [comment]
Thanks for your input, the issue has been reproduced and we've created a public issue to solve this problem.

https://issues.alfresco.com/jira/browse/SEARCH-1899

**Comment by tpage-alfresco** [comment]
The 6.6.5-patched artifacts are now publicly available as is the two later patched versions 6.6.5-patched.1 and 6.6.5-patched.2.  If this doesn't fix your problem then please get back in contact!

Nb. The end-to-end tests project (e2e-test) will still not compile without internal access, but we thought that it might potentially still useful to see the end-to-end test cases that we've automated.  The unit and integration tests should work fine.
